.gitignore

```diff
@@ -15,3 +15,5 @@ debug.test
 site/
 /go-diagrams/
 argo-events
+.swo
+.swp
```

Comment:
Are those needed? Alternatively do you want to ignore all temporary vim files using `*.swp`
and `*.swo`?

Reply:
updated!